api/HTMLAllCollection.json

```diff
@@ -5,10 +5,10 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLAllCollection",
         "support": {
           "chrome": {
-            "version_added": true
+            "version_added": "4"
```

Comment:
`document.all` is marked as supported in Chrome 1. Was it `HTMLCollection` between Chrome 1 and 3? Same basic issue as in https://github.com/mdn/browser-compat-data/pull/7753.

Reply:
Per https://github.com/mdn/browser-compat-data/pull/7753#discussion_r542362347 it was indeed a `HTMLCollection` originally. In that PR we went with just representing when `HTMLAllCollection` was added without adding a pedantic note about it on `document.all`, so let's do the same here.